src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonDocument.MetadataDb.cs

```diff
@@ -127,25 +127,6 @@ internal MetadataDb(int payloadLength)
 #endif
             }
 
-            internal MetadataDb(MetadataDb source, bool useArrayPools)
```

Comment:
Was this used?

Reply:
Nope, although Intellisense falsely reported it was. There's anothor ctor with just the length that is called from one location.